src/domain/sales-channels/pages/details.tsx

```diff
@@ -0,0 +1,458 @@
+  setActiveSalesChannel: (sc: SalesChannel) => void
+}
+
+/**
+ * Sales channels list.
+ */
+function SalesChannelsList(props: SalesChannelsListProps) {
+  const {
+    activeChannelId,
+    openCreateModal,
+    setActiveSalesChannel,
+    salesChannels,
+    filterText,
+    setFilterText,
+  } = props
+
+  return (
+    <div className="col-span-1 rounded-lg border bg-grey-0 border-grey-20 px-8 py-6">
+      <SalesChannelsHeader
+        filterText={filterText}
+        setFilterText={setFilterText}
+        openCreateModal={openCreateModal}
+      />
+      <div>
+        {salesChannels?.map((s) => (
+          <SalesChannelTile
+            salesChannel={s}
+            isDisabled={s.is_disabled}
+            isSelected={activeChannelId === s.id}
+            onClick={() => setActiveSalesChannel(s)}
+          />
+        ))}
+      </div>
+    </div>
+  )
+}
+
+type SalesChannelDetailsHeaderProps = {
+  salesChannel: SalesChannel
+  openUpdateModal: () => void
+  resetDetails: () => void
+  setShowAddProducts: (show: boolean) => void
+}
+
+/**
+ * Sales channels details header.
+ */
+function SalesChannelDetailsHeader(props: SalesChannelDetailsHeaderProps) {
+  const {
+    salesChannel,
+    openUpdateModal,
+    resetDetails,
+    setShowAddProducts,
+  } = props
+
+  const { mutate: deleteSalesChannel } = useAdminDeleteSalesChannel(
+    salesChannel.id
+  )
+
+  const { mutate: updateSalesChannel } = useAdminUpdateSalesChannel(
+    salesChannel.id
+  )
+
+  const [showDelete, setShowDelete] = useState(false)
+
+  const actions = [
+    {
+      label: "Edit general info",
+      icon: <EditIcon size="20" />,
+      onClick: openUpdateModal,
+    },
+    {
+      label: "Add/Edit products",
+      icon: <PlusIcon />,
+      onClick: () => setShowAddProducts(true),
+    },
+    {
+      label: "Delete sales channel",
+      icon: <TrashIcon size={20} />,
+      variant: "danger",
+      onClick: () => setShowDelete(true),
+    },
+  ]
+
+  return (
+    <div className="flex justify-between items-center">
+      <h2 className="font-semibold text-xlarge text-grey-90 mb-4">
+        {salesChannel.name}
+      </h2>
+      <div className="flex justify-between items-center gap-4">
+        <StatusSelector
+          onChange={() =>
+            updateSalesChannel({ is_disabled: !salesChannel.is_disabled })
+          }
+          isDraft={salesChannel.is_disabled}
+          draftState="Disabled"
+          activeState="Enabled"
+        />
+        <Actionables forceDropdown={true} actions={actions} />
+      </div>
+
+      {showDelete && (
+        <DeletePrompt
+          handleClose={() => setShowDelete(false)}
+          onDelete={async () => {
+            deleteSalesChannel()
+            resetDetails()
+          }}
+          confirmText="Yes, delete"
+          successText="Sales channel deleted"
+          text={`Are you sure you want to delete "${salesChannel.name}" sales channel?`}
+          heading="Delete sales channel"
```

Comment:
question: What did we land upon with the capitalization of the header in the delete component, should this also be "Delete Sales Channel"

Reply:
I agree it should be consistent but I would go with "Delete sales channel"

Reply:
The design here has been updated to just reference sales channel as "chanel" lower case